test/blackbox-tests/test-cases/pkg/helpers.sh

```diff
@@ -18,6 +18,14 @@ show_pkg_cookie() {
   $dune internal dump $pkg_root/$1/target/cookie
 }
 
+if [ ! -d mock-opam-repository ]
```

Comment:
Could you make this an explicit function instead?

Reply:
That would mean we have to call it everywhere as well. Is that ok?

Reply:
Yeah, that's how we do it for all other setup functions in the tests.